test/integration/common/StudioActions.kt

```diff
@@ -204,6 +194,19 @@ object StudioActions {
         }
     }
 
+    suspend fun openSchemaWriteTransaction(dbName: String, composeRule: ComposeContentTestRule) {
+        Service.driver.session.tryOpen(dbName, TypeDBSession.Type.SCHEMA)
+        Service.driver.tryUpdateTransactionType(TypeDBTransaction.Type.WRITE)
+
+        clickText(composeRule, Label.SCHEMA.lowercase())
+        clickText(composeRule, Label.WRITE.lowercase())
+
+        waitUntilTrue(composeRule) {
+            Service.driver.session.type == TypeDBSession.Type.SCHEMA &&
+                    Service.driver.session.transaction.type == TypeDBTransaction.Type.WRITE
+        }
```

Comment:
After a discussion with James and stepping through in a debugger, we (re)discovered that `tryOpen` not only opens the session, but also assigns the current database in the UI. We have to do this because the database dropdown list is a floating window and invisible to the test suite. Instead, we're introducing this little helper for schema-write sessions.